# fixes compile error: for some reason, ifort fails on this (possibly compiler bug)
# As a solution, we always display the warning
# By Ward Poelmans <wpoely86@gmail.com>
diff -ur c37b2.orig/source/squantm/sqnt_qm2_ewald.src c37b2/source/squantm/sqnt_qm2_ewald.src
--- c37b2.orig/source/squantm/sqnt_qm2_ewald.src	2012-07-17 01:08:32.000000000 +0200
+++ c37b2/source/squantm/sqnt_qm2_ewald.src	2013-12-03 14:11:46.945196858 +0100
@@ -1876,7 +1878,7 @@
 ##IF COLFFT (colfft)
       !==================COLUMN FFT METHOD ==============================
 ##IF QUANTUM MNDO97 SQUANTM GAMESS GAMESSUK QCHEM QTURBO G09
-      if(LQMEWD) call wrndie(-5,'<QM_PME_MM_POT>','QM/MM-PME do not support COLFFT.')
+      call wrndie(1,'<QM_PME_MM_POT>','QM/MM-PME do not support COLFFT.')
 ##ENDIF
 ##ELSE      (colfft)
       !.ab.Fix: xnsymm should be set (not 0 !).
@@ -2146,7 +2148,7 @@
 ##IF COLFFT (colfft)
       !==================COLUMN FFT METHOD ==============================
 ##IF QUANTUM MNDO97 SQUANTM GAMESS GAMESSUK QCHEM QTURBO G09
-      if(LQMEWD) call wrndie(-5,'<PME_QMMM>','QM/MM-PME do not support COLFFT.')
+      call wrndie(1,'<PME_QMMM>','QM/MM-PME do not support COLFFT.')
 ##ENDIF
 ##ELSE      (colfft)
 
